docs/DeveloperGuide.md

```diff
@@ -1,11 +1,82 @@
 # Developer Guide
 
-## Design & implementation
+## Design
+
+### Architecture
+
+
+### Utilities Component
+The Utilities component contains the main classes that run the main functions of Mojo.
+
+![Utility diagram](UML diagrams/Utilities.jpeg)
+
+The Utilities Class Diagram given above shows how the classes in the Utilities component interact with each other and classes from other component.
+
+The Utilities Component consists for 3 classes.
+
+* `Parser`: Breaks down user input into relevant objects.
+* `Storage`: Reads data from, and writes data to, the local disk.
+* `Ui`: Handles the input and output of the application.
+
+
+## implementation
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+![Architect class diagram](UML diagrams/OverviewClassDiagram.png)
+![Architect sequence diagram](UML diagrams/OverviewSequenceDiagram.png)
```

Comment:
![image](https://user-images.githubusercontent.com/56064050/113082860-cc9cf280-920d-11eb-9d18-9b4f67a5beb0.png)

Is the placement for the Object Deletion notation for the Command Object correct? Would it be clearer if it is placed slightly below the activation bar?